recipes/azure-sdk-for-cpp/all/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.15)
+project(cmake_wrapper)
+
+foreach(sdk ${BUILD_LIST})
+    if(${sdk} STREQUAL azure-core)
+        # Always build Core
+        add_subdirectory("src/sdk/core")
+    elseif(${sdk} STREQUAL azure-storage-common)
```

Comment:
A few lines explaining why this CMakeLists.txt needs to be injected in the build would be great.